codespell_lib/data/dictionary.txt

```diff
@@ -13301,6 +13453,9 @@ samller->smaller
 sanctionning->sanctioning
 sandwhich->sandwich
 Sanhedrim->Sanhedrin
+sanitiation->sanitisation
+sanitised->sanitized
```

Comment:
AE <-> BE conversion

Reply:
I don't understand

Reply:
American English/British English, remove it and stick a note in #103 .

Reply:
Removed